workshops/full-stack_todo_app/README.md

```diff
@@ -0,0 +1,749 @@
+
+![Realtime Database](https://cloud-9uzz77u17.vercel.app/2database.png)
+
+Firebase initially allows anyone to read and write the database for 30 days. You can edit the rules anytime in the `Rules` tab.
+
+![Rules](https://cloud-9uzz77u17.vercel.app/1database_rules.png)
+
+Now that we have a database let's create a Todo App using the database.
+
+## CRUD Operations
+
+According to [wikipedia](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete)...
+
+> "In computer programming, create, read, update, and delete (CRUD) are the four basic functions of persistent storage. Alternate words are sometimes used when defining the four basic CRUD functions, such as retrieval instead of reading, modifying instead of update, or destroy instead of delete.<br/>
+ CRUD is also sometimes used to describe user interface conventions that facilitate viewing, searching, and changing information, often using computer-based forms and reports. The term was likely first popularized by James Martin in his 1983 book Managing the Data-base Environment. The acronym may be extended to CRUDL to cover the listing of large data sets which bring additional complexity such as pagination when the data sets are too large to hold easily in memory."
+
+CRUD stands for Create, Read, Update, and Delete.
+
+These are the four most basic operations that can be performed with most traditional database systems, and they are the backbone for interacting with any database. 
+
+![CRUD](https://cloud-9uzz77u17.vercel.app/0crud_operations.png)
+
+## Creating Todo App
+
+Let's create the interface of the Todo App to perform CRUD operations.
+
+Create a new folder and name it as `components`. All of our Todo App components will be in this folder.
+
+### Inputting a Todo
+
+Create a new file in the `components` folder and name it `Input.js`. Import `React` into the component and create the functional component, and export it.
+
+```jsx
+import React from 'react'
+
+function Input(){
+  // code goes here
+  }
+ 
+ export default Input
+```
+
+Also, import the `Firebase.js` file you created before at the top to use Firebase inside this component.
+
+```jsx
+import firebse from '../Firebase'
+```
+
+Create a `div` with a class of `input` inside the component.  
+
+```jsx
+import React from 'react'
+import firebase from '../Firebase'
+
+function Input(){
+  return(
+    <div className="input">
+      // code goes here
+    </div>
+  )
+}
+
+export default Input
+```
+
+Add an `input` and a button `Add Todo` with a class of `add-btn` inside the `div` element.
+
+```html
+<div className="input">
+  <input type="text" placeholder="Enter a Todo..." />
+  <button className="add-btn">
+  <i className="fa fa-plus-circle" aria-hidden="true"></i><span id="btn-text">Add Todo</span> 
+  </button>
+</div>
+```
+
+Here, `<i className="fa fa-plus-circle">` is the [plus icon](https://fontawesome.com/v4.7.0/icon/plus-circle) from [font awesome](https://fontawesome.com/v4.7.0/icons/) for styling. 
```

Comment:
can u explain where we imported font-awesome and sme stuff about it? coz if people directly see font-awesome being used without any information, they'll get confused